ExecutionTimer unit test fails intermittently for Windows GitHub Action

### Summary

The `ExecutionTimerTest.test_execute` unit test method is failing occasionally for the Windows GitHub Action workflow. Re-running the test eventually causes it to pass. This seems to be affecting Windows only.

### Steps To Reproduce

Requires investigation to see what is causing the failures.

### Expected Behaviour

Passes consistently

### Current Behaviour

Fails intermittently

### Context

Current main

### Failure Logs

```
self = <mantidimaging.core.utility.test.execution_timer_test.ExecutionTimerTest testMethod=test_execute>

    def test_execute(self):
        t = ExecutionTimer()
        self.assertEqual(t.total_seconds, None)
        self.assertEqual(str(t), 'Elapsed time: unknown seconds')
    
        with t:
            self.assertEqual(t.total_seconds, None)
            self.assertEqual(str(t), 'Elapsed time: unknown seconds')
    
            time.sleep(0.1)
    
>       self.assertAlmostEqual(t.total_seconds, 0.1, delta=0.05)
E       AssertionError: 0.5644240379333496 != 0.1 within 0.05 delta (0.46442403793334963 difference)

mantidimaging\core\utility\test\execution_timer_test.py:22: AssertionError
```

--- a/mantidimaging/core/utility/execution_timer.py
+++ b/mantidimaging/core/utility/execution_timer.py
@@ -15,9 +15,9 @@ class ExecutionTimer(object):
         self.time_end = None
 
     def __str__(self):
-        prefix = '{}: '.format(self.msg) if self.msg else ''
+        prefix = f'{self.msg}: ' if self.msg else ''
         sec = self.total_seconds
-        return '{}{} seconds'.format(prefix, sec if sec else 'unknown')
+        return f'{prefix}{sec if sec else "unknown"} seconds'
 
     def __enter__(self):
         self.time_start = time.time()

--- a/mantidimaging/core/utility/test/execution_timer_test.py
+++ b/mantidimaging/core/utility/test/execution_timer_test.py
@@ -15,7 +15,6 @@ class ExecutionTimerTest(unittest.TestCase):
 
         with t:
             self.assertEqual(t.total_seconds, None)
-            self.assertEqual(str(t), 'Elapsed time: unknown seconds')
 
             time.sleep(0.1)
 

